### DISCUSSION TITLE
過去の面白記事の英訳

### DISCUSSION THREAD

**Discussion by nanophate** [rubima/magazine.rubyist.net#391] [要望・アイデア]
@gunjisatoshi 日本語だけで面白い記事が書かれてるのは勿体ない。英語に訳してもっとたくさんの人たちに読んでもらいたいです。

**Comment by gunjisatoshi** [comment]
@nanophate ご提案ありがとうございます！（先日の沖縄 Ruby 会議 01 ではお世話になりました！）

具体的な進め方については相談させてください！（とりあえずこの Issue で議論しましょう！）

**Comment by nanophate** [comment]
@gunjisatoshi ありがとうございます。（こちらこそ、沖縄 Ruby 会議 01ではお世話になりました。）
これから、よろしくお願いいたします。
